test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorHostTest.cs

```diff
@@ -481,6 +494,60 @@ public override IReadOnlyList<ChunkTree> GetInheritedChunkTrees([NotNull] string
             }
         }
 
+        // Normalizes the newlines in different OS platforms.
+        private class MvcRazorHostWithNormalizedNewLine : MvcRazorHost
+        {
+            public MvcRazorHostWithNormalizedNewLine(IChunkTreeCache codeTreeCache)
+                : base(codeTreeCache)
+            { }
+
+            public override CodeGenerator DecorateCodeGenerator(
+                CodeGenerator incomingBuilder,
+                CodeGeneratorContext context)
+            {
+                base.DecorateCodeGenerator(incomingBuilder, context);
+
+                return new TestCSharpCodeGenerator(
+                    context,
+                    DefaultModel,
+                    "Microsoft.AspNet.Mvc.Razor.Internal.RazorInjectAttribute",
+                    new GeneratedTagHelperAttributeContext
+                    {
+                        ModelExpressionTypeName = ModelExpressionType,
+                        CreateModelExpressionMethodName = CreateModelExpressionMethod
+                    });
+            }
+
+            protected class TestCSharpCodeGenerator : MvcCSharpCodeGenerator
+            {
+                private readonly GeneratedTagHelperAttributeContext _tagHelperAttributeContext;
+
+                public TestCSharpCodeGenerator(CodeGeneratorContext context,
+                                             string defaultModel,
+                                             string activateAttribute,
+                                             GeneratedTagHelperAttributeContext tagHelperAttributeContext)
+                    : base(context, defaultModel, activateAttribute, tagHelperAttributeContext)
+                {
+                    _tagHelperAttributeContext = tagHelperAttributeContext;
+                }
+
+                protected override CSharpCodeWriter CreateCodeWriter()
```

Comment:
Adding comment here to expose https://github.com/aspnet/Mvc/pull/2771/files#r33731566 comment so it doesn't stay hidden once code review comments are addressed.